pennylane_lightning/core/src/simulators/lightning_gpu/catalyst/tests/Test_LightningGPUSimulator.cpp

```diff
@@ -678,6 +678,95 @@ TEST_CASE("LightningGPUSimulator::GateSet", "[GateSet]") {
         REQUIRE(LGPUsim->CacheManagerInfo() == expected);
     }
 
+    // ============= Controlled operations =============
```

Comment:
Do we want to remove this line?

Reply:
This is in a big test and helpful to mark the specific areas. Also similar to [this line ](https://github.com/PennyLaneAI/pennylane-lightning/blob/422fc7966a001496c47ea8b3776ce65cc0062f0e/pennylane_lightning/core/src/simulators/lightning_gpu/catalyst/tests/Test_LightningGPUSimulator.cpp#L318) in the file